ext/storage_sources/local_store/local_store.c

```diff
@@ -247,8 +247,16 @@ local_delay(LOCAL_STORAGE *local)
         WT_VERBOSE_LS(local,
           "Artificial delay %" PRIu32 " milliseconds after %" PRIu64 " object flushes\n",
           local->delay_ms, local->object_flushes);
+        /*
+         * tv_usec has type suseconds_t, which is signed (hence the
+         * s), but ->delay_ms is unsigned. In both gcc8 and gcc10 with
+         * -Wsign-conversion enabled (as we do) this causes a spurious
```

Comment:
Could you please format this so the `-` character is not at the beginning of this line? So for example, move the `with` down to this line.

If you run `dist/s_all` after doing that, it should reformat this comment to fill out the full 100 character line length. The reason it didn't do it here is that it avoids doing it with lines starting with `-` since that often represents a set of bullet points in a comment block.

Reply:
Oops. Had no idea. Fixed...